README.md

```diff
@@ -11,6 +11,35 @@ Implements all [recommended best practises](https://developer.github.com/v3/guid
 
 ## Usage
 
+<table>
+<tbody valign=top align=left>
+<tr><th>
+Browsers
+</th><td width=100%>
+Load <code>@octokit/plugin-throttling</code> directly from <a href="https://unpkg.com">unpkg.com</a>
+        
+```html
+<script type="module">
+import pluginThrottling from "https://unpkg.com/@octokit/plugin-throttling";
+</script>
+```
+
+</td></tr>
+<tr><th>
+Node
+</th><td>
+
+Install with <code>npm install @octokit/plugin-throttling</code>
+
+```js
+const pluginThrottling = require("@octokit/plugin-throttling");
+// or: import pluginThrottling from "@octokit/plugin-throttling";
+```
+
+</td></tr>
+</tbody>
+</table>
```

Comment:
That looks good!

Once we have `@octokit/rest` available on unpkg.com we should update these examples to also load `@octokit/rest` from there. I’ll make a follow up issue :)